contracts/verifiers/ZKPVerifierBase.sol

```diff
@@ -237,21 +245,7 @@ abstract contract ZKPVerifierBase is IZKPVerifier, ContextUpgradeable {
         uint256 startIndex,
         uint256 length
     ) public view returns (IZKPVerifier.ZKPRequest[] memory) {
-        ZKPVerifierStorage storage s = _getZKPVerifierStorage();
-        (uint256 start, uint256 end) = ArrayUtils.calculateBounds(
-            s._requestIds.length,
-            startIndex,
-            length,
-            REQUESTS_RETURN_LIMIT
-        );
-
-        IZKPVerifier.ZKPRequest[] memory result = new IZKPVerifier.ZKPRequest[](end - start);
-
-        for (uint256 i = start; i < end; i++) {
-            result[i - start] = s._requests[s._requestIds[i]];
-        }
-
-        return result;
+        revert("Not implemented in this version");
```

Comment:
You should not remove it from base contract. Only from universal verifier

Reply:
Not sure if it is possible without defining function as virtual.
And probably this will change its signature

Reply:
Not possible without `virtual` function, but it's not changing function signature nor ABI. (these virtual/override are not part of the signature). Reverted in Universal verifier